pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -29,6 +29,7 @@ import (
 )
 
 func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadBalancerName string, listeners []*elb.Listener, subnetIDs []string, securityGroupIDs []string, internalELB bool) (*elb.LoadBalancerDescription, error) {
+	glog.V(2).Infof("ensureLoadBalancer(%v, %v, %v, %v, %v, %v)", namespacedName, loadBalancerName, listeners, subnetIDs, securityGroupIDs, internalELB)
```

Comment:
This change is causing the merge conflict (I think), and I think is unnecessary if you have the other one.  How about you remove this change, and then I'll LGTM the PR (i.e. the other log statement).